app/src/main/java/kr/ac/konkuk/gdsc/gdscsuyeon/ui/home/TodoAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package kr.ac.konkuk.gdsc.gdscsuyeon.ui.home
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import kr.ac.konkuk.gdsc.gdscsuyeon.R
+import kr.ac.konkuk.gdsc.gdscsuyeon.data.Todo
+import kr.ac.konkuk.gdsc.gdscsuyeon.databinding.RowBinding
+
+class TodoAdapter(var todos: ArrayList<Todo>) : RecyclerView.Adapter<TodoAdapter.ViewHolder>(){
```

Comment:
```suggestion
class TodoAdapter(val todos: ArrayList<Todo>) : RecyclerView.Adapter<TodoAdapter.ViewHolder>(){
```

var, ArrayList 모두 가변성질을 가져서 하나 정도는 불변으로 돌리시는 것을 권장드릴게요!